Src/Base/AMReX_Geometry.cpp

```diff
@@ -397,4 +400,55 @@ Geometry::growPeriodicDomain (int ngrow) const noexcept
     return b;
 }
 
+void
+Geometry::computeRoundoffDomain ()
+{
+    using PReal = ParticleReal;
+    roundoff_domain = prob_domain;
+    for (int idim = 0; idim < AMREX_SPACEDIM; ++idim)
+    {
+        int ilo = Domain().smallEnd(idim);
+        int ihi = Domain().bigEnd(idim);
+        Real plo = ProbLo(idim);
+        Real phi = ProbHi(idim);
+        Real deltax = CellSize(idim);
+
+#ifdef AMREX_SINGLE_PRECISION_PARTICLE
+        PReal tolerance = std::max(1.e-8*deltax, 1.e-14*phi);
+#else
+        PReal tolerance = std::max(1.e-4*deltax, 1.e-10*phi);
+#endif
```

Comment:
Seems that the lines should be switched.

Reply:
Yes, that was ill-formed. Thanks.